src/System.Management.Automation/engine/hostifaces/ConnectionBase.cs

```diff
@@ -43,7 +43,7 @@ protected RunspaceBase(PSHost host)
             {
                 throw PSTraceSource.NewArgumentNullException("host");
             }
-            InitialSessionState = InitialSessionState.CreateDefault2();
+            InitialSessionState = InitialSessionState.CreateDefault();
```

Comment:
Are we sure that we want to use CreateDefault() and load all commands/functions?  Or CreateDefault2() where just Core commands are loaded (and is faster)?  Please add a comment to justify which default we use.

Reply:
This is where I need some expertise to mimic the previous RunspaceConfiguration behavior.  @lzybkr @daxian-dbw ?